src/Resources/config/listener.yml

```diff
@@ -6,6 +6,7 @@
 #
 # - 30: ContainerScopeListener
 # - 28: RefererIdListener
+# - 27: InsecureInstallationListener
```

Comment:
As long as the listener priorities are not reworked, we should keep one number in between each listener priority. This means it should be:

```
- 30: ContainerScopeListener
- 28: RefererIdListener
- 26: InsecureInstallationListener
- 24: InitializeSystemListener
- 22: ToggleViewListener
- 20: OutputFromCacheListener
```